src/main/scala/com/exasol/cloudetl/helper/ParquetSchemaConverter.scala

```diff
@@ -0,0 +1,143 @@
+package com.exasol.cloudetl.helper
+
+import java.util.Locale.ENGLISH
+
+import com.exasol.cloudetl.data.ExaColumnInfo
+
+import org.apache.parquet.schema.LogicalTypeAnnotation._
+import org.apache.parquet.schema.MessageType
+import org.apache.parquet.schema.PrimitiveType.PrimitiveTypeName
+import org.apache.parquet.schema.Type
+import org.apache.parquet.schema.Type.Repetition
+import org.apache.parquet.schema.Types
+
+/**
+ * A class that converts Exasol column names to Parquet schema.
+ *
+ * @param isLowercaseSchemaEnabled a boolean flag to make Parquet schema fields lowercase
+ */
+final case class ParquetSchemaConverter(isLowercaseSchemaEnabled: Boolean) extends JavaClassTypes {
+  import ParquetSchemaConverter._
+
+  /**
+   * Given the Exasol column information returns Parquet [[org.apache.parquet.schema.MessageType]].
+   *
+   * @param columns a list of Exasol column information
+   * @param schemaName a Parquet schema name
+   * @return a Parquet message type
+   */
+  def createParquetMessageType(columns: Seq[ExaColumnInfo], schemaName: String): MessageType = {
+    val types = columns.map(columnToParquetField(_))
+    new MessageType(schemaName, types: _*)
+  }
+
+  private[this] def columnToParquetField(columnInfo: ExaColumnInfo): Type = {
+    val columnName = convertIdentifier(columnInfo.name)
+    getParquetType(columnName, columnInfo)
+  }
+
+  private[this] def convertIdentifier(columnName: String): String = {
+    val convertedName = columnName.replace("\"", "")
+    if (isLowercaseSchemaEnabled) {
+      convertedName.toLowerCase(ENGLISH)
```

Comment:
Is English the only language allowed in identifiers?

Reply:
No this is more about how upper case and lower case should be performed. We want standard English way of converting cases.

For example, `TITLE` to lowercase in default Turkish locale would result in `tıtle`, notice not dotted `i` in the result, because Turkish alphabet contains both `i` and `ı` letters, with and without dots.

For info here: https://stackoverflow.com/questions/11063102/using-locales-with-javas-tolowercase-and-touppercase